### PR TITLE
Update and rename sky-ticket.rb to wow-tv.rb

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Sky ticket rebranded to wow tv, not much else changed…

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
Hmm, not completely sure how to handle renames…

File: Casks/wow-tv.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  homepage "https://wowtv.de/"
+
+  livecheck do
+    url "https://web.static.nowtv.com/watch/player/wowtv/de/latest/update.json"
+    strategy :page_match do |page|
+      JSON.parse(page)["platforms"]["darwin"]["version"]
+    end
+  end
+
+  app "Sky Ticket.app"
```
Comment (p-linnane) [inline Casks/wow-tv.rb:18]:
```suggestion
  app "WOW.app"
```

**Comment by ChrisChinchilla** [comment]
🤦 Thanks @p-linnane !

**Comment by miccal** [comment]
This re-name should not cause too many issues:
```
==> Analytics
install: 7 (30 days), 31 (90 days), 101 (365 days)
```

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**